case-study-azure-devops.md

```diff
@@ -0,0 +1,416 @@
+
+In your builds and releases, you can use the packages from the Package Management feed by specifying the feed in your build or release definition. Azure DevOps will then download the packages and make them available to your build agents or release agents.
+
+Package Management in Azure DevOps helps to centralize your package management and improve collaboration and reuse within the organization. The Package Management ensures that your builds and releases use consistent and up-to-date packages.
+
+*Reference [[12](#references)]*
+
+### Code Review and Feedback
+
+Azure DevOps offers a code review and feedback system similar to other popular platforms, like GitHub. You create a pull request and on that request you can add team members to look at the code and review it.
+
+*Reference [[13](#references)]*
+
+### Integrated Analytics
+
+Azure DevOps provides Integrated Analytics which is a feature that helps you analyze and visualize data from various sources within Azure DevOps. Integrated Analytics uses Azure Boards and Azure DevOps data to provide insights into your work items, code changes and pipelines.
+
+With Integrated Analytics you can create and share reports and dashboards that visualize data from Azure DevOps in meaningful ways. For example, you can create a dashboard that shows the status of your work items, the number of code changes made in a given time period or the success rate of your pipelines.
+
+To create a report or dashboard you can use the built-in report authoring tools in Azure DevOps, or you can use Power BI, which is a data visualization tool provided by Microsoft. To create a report or dashboard in Power BI, you can connect to the Azure DevOps data source and use the data to create charts, graphs, and other visualizations.
+
+Integrated Analytics in Azure DevOps helps you make data-driven decisions by giving you access to rich data visualizations and insights into your work. With Integrated Analytics, you can monitor the progress of your work and identify areas where improvements can be made.
+
+*Reference [[14](#references)]*
+
+## **Strengths**
+
+Azure DevOps have lots of strengths, but it's important to keep in mind that the strengths of the platform may vary depending on the specific needs of an organization and that the strengths of the platform may outweigh these weaknesses for some use cases. 
+
+**Here is some of the strengths of Azure DevOps**
+
+1. Its really easy to integrate with the widley used platform GitHub. In fact 2018 Microsoft bought GitHub for $7.5 billion, so the connection between the popular code-repository service used by many developers and large companies and Azure DevOps is really easy to setup and use. The boards that Azure provides have more options to offer and look better then in the project boards that Github provides. So to be able to use the boards that Azure DevOps offer combined with Github repo is great.
```

Comment:
```suggestion
1. Its really easy to integrate with the widley used platform GitHub. In fact 2018 Microsoft bought GitHub for $7.5 billion, so the connection between the popular code-repository service used by many developers and large companies and Azure DevOps is really easy to setup and use. The boards that Azure provides have more options to offer and look better than in the project boards that Github provides. So to be able to use the boards that Azure DevOps offer combined with Github repo is great.
```

Reply:
This is quite subjective, it would be nice to be a bit more objective. "Looks better" and "is great" are not technical assessments